src/Http/Routing/src/Matching/JumpTableBuilder.cs

```diff
@@ -85,11 +86,14 @@ public static JumpTable Build(int defaultDestination, int exitDestination, (stri
         }
 
         // Use the ILEmitTrieJumpTable if the IL is going to be compiled (not interpreted)
-        if (RuntimeFeature.IsDynamicCodeCompiled)
+        return MakeILEmitTrieJumpTableIfSupported(defaultDestination, exitDestination, pathEntries, fallback);
+
+        [UnconditionalSuppressMessage("AOT", "IL3050", Justification = "Guarded by IsDynamicCodeCompiled")]
```

Comment:
Can you add a comment referencing https://github.com/dotnet/linker/issues/2715 here? That way this change can be reverted once that bug is fixed in the analyzer.

Reply:
We need a follow up bug for these things